lib/Room.js

```diff
@@ -270,10 +270,30 @@ class Room extends EnhancedEventEmitter
 				};
 
 				// TODO: The app should provide his own RTP capabilities.
-				const consumerRtpParameters =
+				let consumerRtpParameters =
 					ortc.getConsumerRtpParameters(
 						producer.consumableRtpParameters, this.rtpCapabilities);
 
+				// Remove ffmpeg-proxy from capabilities unless sendOldNack is set
+				if (options.sendOldNack !== true)
+				{
+					consumerRtpParameters.codecs.forEach((codec) => {
+						if (codec.rtcpFeedback !== undefined && Array.isArray(codec.rtcpFeedback)) 
+						{
+							let dbgfeed = codec.rtcpFeedback.filter((val) => val.type === 'ffmpeg-proxy'); // TODO: remove before checkin
+							if (dbgfeed.length > 0)
+								logger.debug('"createRtpStreamer()" has ffmpeg-proxy rtcp-fb: %s', dbgfeed);
+
+							let rtcpFeedback = codec.rtcpFeedback.filter((val) => val.type !== 'ffmpeg-proxy');
+							codec.rtcpFeedback = rtcpFeedback;
+						}
+					});
+				}
+				else
+				{
+					logger.debug('"createRtpStreamer()" no changes in ffmpeg-proxy rtcp-fb '); // TODO: remove before checkin
```

Comment:
don't forget to remove?

Reply:
Cleaned up yesterday after testing, thanks!